openslides/users/rest_auth.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.authentication import BaseAuthentication
+
+from openslides.config.api import config
+
+
+# TODO: Move this into the file 'auth' after fixing circular imports
```

Comment:
Solve this by importing the Permission class from django.contrib.auth and not from models